test/SmokeTests/OOProcSmokeTests/DotNetIsolated/DotNetIsolated.csproj

```diff
@@ -11,14 +11,14 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.19.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.0.13" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.14.1" OutputItemType="Analyzer" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.16.0-preview1" OutputItemType="Analyzer" />
```

Comment:
This is the SDK version I have in limited preview (hence the new `AzureFunctionsTempStaging` feed). It overhauls the functions inner-build steps. The important part for us here is it will explicitly pass nuget restore sources from outer build to inner build.

Reply:
nit: let's add this information as a comment in the csproj itself. Otherwise, it's a bit confusing why we'd reference a `preview` package like this